### DISCUSSION TITLE
Hm, License!? Ooookay, BUT...

### DISCUSSION THREAD

**Discussion by Tezei** [community/community#120108] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello!
I just got a license notification surprise for @PySimpleGUI.

Okay, I understand.

One of the major advantages that people are using Python is that we have possibility to use free modules and packages for a lot of things. So, if the future is like this, we just will switch the language.
 
BUT! What about legacy code in which may be invested a lot during this "license situations". So a lot of people should invest to rewrite the code if they are not agree on your terms.

So, what is happening with the legacy code?
Please advise!

Thanks in advance!

**Comment by ECOM-Engineering** [comment]
I have the same question!
